packages/accordions/package.json

```diff
@@ -22,7 +22,9 @@
   "sideEffects": false,
   "types": "dist/typings/index.d.ts",
   "dependencies": {
-    "@zendeskgarden/container-utilities": "^0.5.1",
+    "@zendeskgarden/container-accordion": "^1.0.0",
+    "@zendeskgarden/container-utilities": "^0.5.3",
+    "@zendeskgarden/react-typography": "^8.6.0",
```

Comment:
Is this only used in the markdown examples? This should be a `devDependency` if so.

Reply:
Do we even add `devDependency` for other markdown uses? For example we have `Grid`, `Well`, and `Input` all over the demos without needing to import.

Reply:
@jzempel it looks like this isn't needed. I'll remove this and give it a try, and update accordingly.